dali/kernels/audio/mel_scale/mel_filter_bank_gpu_test.cc

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <random>
+#include <tuple>
+#include <vector>
+#include "dali/test/tensor_test_utils.h"
+#include "dali/test/test_tensors.h"
+#include "dali/kernels/audio/mel_scale/mel_filter_bank_gpu.h"
+#include "dali/kernels/audio/mel_scale/mel_filter_bank_test.h"
+#include "dali/kernels/kernel_manager.h"
+
+namespace dali {
+namespace kernels {
+namespace audio {
+namespace test {
+
+
+using TestBase = ::testing::TestWithParam<
+    std::tuple<std::vector<TensorShape<4>>, /* data_shape */
```

Comment:
Please align the comments for readability. Anyway, lint would likely protest at the lack of two spaces between code and comment. Also: single rest-of-line block comments are obsolete and discouraged.

Reply:
done